From 165ae6edc64ad6fa2129bedcac2837d24773e31f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 26 Feb 2020 23:34:41 +0100
Subject: [PATCH] kernel-install: strip BOOT_IMAGE= from kernel options

https://bugzilla.redhat.com/show_bug.cgi?id=1716164.
(cherry picked from commit e60228bf68427c0c0f96f816ee9124fd39585038)
(cherry picked from commit 2095b6a279cfe6e9688c5c357d550289dfcd8e8d)
---
 src/kernel-install/90-loaderentry.install | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/kernel-install/90-loaderentry.install b/src/kernel-install/90-loaderentry.install
index 47a0cd224a..133378da6b 100644
--- a/src/kernel-install/90-loaderentry.install
+++ b/src/kernel-install/90-loaderentry.install
@@ -55,6 +55,7 @@ else
     read -r -d '' -a line < /proc/cmdline
     for i in "${line[@]}"; do
         [[ "${i#initrd=*}" != "$i" ]] && continue
+        [[ "${i#BOOT_IMAGE=*}" != "$i" ]] && continue
         BOOT_OPTIONS+=("$i")
     done
 fi
